src/utils/index.js

```diff
@@ -24,3 +24,25 @@ export class FastFillStyle {
     }
   }
 }
+
+/**
+ * Perform a simple shallow object equality check.
+ */
+export function objectShallowEquals(a: Object, b: Object): boolean {
```

Comment:
I felt that it was simpler to write this function than to include an additional dependency the project. I think we previously had some lodash functions, but now I'm not seeing any, so I didn't want to add additional bytes to the bundle for something so small.